src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/TopNav.kt

```diff
@@ -0,0 +1,18 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.page.IssueCreateDialog
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+
+class TopNav(val driver: WebDriver) {
+    internal fun openIssueCreateDialog(): IssueCreateDialog {
```

Comment:
Why internal?

Reply:
`internal` is a grey area in API. Java compiler will ignore it. If you use JPT from Java and IDE other than IntelliJ you can unconsciously misuse it as API.

Reply:
The original method was internal. I don't mind either way.

Reply:
Also, IssueCreateDialog is internal. Do you want to make these public?

Reply:
`internal` is a mess. 
I'd bite the bullet and make it `public`, pulling in `IssueCreateDialog` along with it.

Reply:
Come to think of it - what's the purpose of a `TopNav` API, which can only check `isEnabled`. If we want to expose `TopNav`, let's expose it usefully - with more methods. If not, then we hide it from API.
So either full public or hide it outside of `api` package.